src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+.fieldset {
+  margin-bottom: 20px;
+  padding-bottom: 0;
+}
+
+.form-field {
+  display: block;
+  margin-bottom: 10px  ;
+}
+
+.form-field2 {
```

Comment:
do you need this class?

Reply:
@mykhalenych Yes,without this class, my options start on a new line 
https://user-images.githubusercontent.com/124716724/223778509-a105e94e-18a5-4d79-a5ad-6f172c066b65.png